pkg/features/feature.go

```diff
@@ -0,0 +1,37 @@
+package features
+
+import (
+	"k8s.io/component-base/featuregate"
+	"k8s.io/klog/v2"
+)
+
+const (
+	// Every feature gate should add method here following this template:
+	//
+	// // owner: @username
+	// // alpha: v1.X
+	// MyFeature featuregate.Feature = "MyFeature"
+
+	// ClusterClaim will manage the cluster-claim resources in the managed cluster. Disabling the
+	// the feature-gate can make the spoke-agent run without installing cluster-claim CRDs in the
+	// managed cluster.
+	ClusterClaim featuregate.Feature = "ClusterClaim"
+)
+
+var (
+	// DefaultMutableFeatureGate is made up of multiple mutable feature-gates.
+	DefaultMutableFeatureGate featuregate.MutableFeatureGate = featuregate.NewFeatureGate()
+)
+
+func init() {
+	if err := DefaultMutableFeatureGate.Add(defaultRegistrationFeatureGates); err != nil {
+		klog.Fatalf("Unexpected error: %v", err)
+	}
+}
+
+// defaultRegistrationFeatureGates consists of all known ocm-registration
+// feature keys.  To add a new feature, define a key for it above and
+// add it here.
+var defaultRegistrationFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
+	ClusterClaim: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
```
 --feature-gates mapStringBool              A set of key=value pairs that describe feature gates for alpha/experimental features. Options are:
                                                 AllAlpha=true|false (ALPHA - default=false)
                                                 AllBeta=true|false (BETA - default=false)
                                                 ClusterClaim=true|false (BETA - default=true)
```

we have to make this beta otherwise it wont be configured via command-lines @qiujian16

Reply:
it is ok to make this as beta.